apps/marketplace/components/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,202 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ClosedDate from 'shared/ClosedDate'
+import { OpportunitiesPagination } from './OpportunitiesPagination'
+import styles from './Opportunities.scss'
+
+const mapOpenTo = val => {
+  const vals = {
+    allSellers: 'all',
+    someSellers: 'selected',
+    oneSeller: 'one'
+  }
+  return val in vals ? vals[val] : val
+}
+
+const mapStates = val => {
+  const vals = {
+    'Australian Capital Territory': 'ACT',
+    'New South Wales': 'NSW',
+    Victoria: 'VIC',
+    Queensland: 'QLD',
+    'South Australia': 'SA',
+    'Western Australia': 'WA',
+    'Northern Territory': 'NT',
+    Tasmania: 'TAS'
+  }
+  return val in vals ? vals[val] : val
+}
+
+export class Opportunities extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      curPage: props.page
+    }
+    this.setCurrentPage = this.setCurrentPage.bind(this)
+  }
+
+  setCurrentPage(page) {
+    this.setState({
+      curPage: page
+    })
+  }
+
+  getPageCount() {
+    return Math.ceil(this.props.opportunities.length / this.props.limit)
+  }
+
+  render() {
+    const offset = (this.state.curPage - 1) * this.props.limit
+    const opportunities = this.props.opportunities.slice(offset, offset + this.props.limit)
+
+    return (
+      <div className={styles.container}>
+        <article className="opportunities-page" role="main">
+          {opportunities.length === 0 && (
+            <div className="row">
+              <div className="col-xs-12">There are no opportunities to show that match your filter.</div>
+            </div>
+          )}
+          {opportunities.length > 0 && (
+            <div className={`${styles.tableDesktop} ${styles.hideMobile}`} role="table" aria-label="Opportunities">
```

Comment:
do accessibility team have an opinion on the aria tags?

Reply:
I don't think we've heard back on this page yet. I did test using NVDA and I could only get it to announce the table name and column headers (when selecting a cell) using the aria tags, when using div based tables.